run.py

```diff
@@ -110,6 +110,7 @@ async def skip(ctx):
 
         await ctx.send('Now playing: {}'.format(player.title))
         del(queue[0])
+        await client.change_presence(activity=discord.Activity.listening, name=('{}'.format(player.title)))
     else:
         await ctx.send("There's none music in the queue")
```

Comment:
*  A string que a Seulgi responde fica melhor com : "There's no songs in the queue" 
*  Deletar a música atual do array depois de trocar a atividade da Seulgi (inverter a 112 e 113)